CONTRIBUTING.md

```diff
@@ -17,7 +17,7 @@ In this example we are using [virtualenvwrapper](https://virtualenvwrapper.readt
 ```bash
 (host)$ pip install virtualenvwrapper
 (host)$ mkvirtualenv ansible-runner
-(host)$ pip install -e .
+(host)$ pip install -e . -r test/requirements.txt -c test/constraints.txt
```

Comment:
I don't think this makes sense to do. Tox is the thing that will install these requirements into the virtual environments that _it_ creates. You shouldn't need to install the requirements into the outer environment. It's unnecessary since they won't even be used for the tests when using tox.

Reply:
I went back and forth on this. The reason I ended up adding it is because `test/requirements.txt` contains `tox` (and `constraints.txt` constrains the version).

We would need to at least replace `-r test/requirements.txt` with `tox`. That may make more sense.

Reply:
I also found that `tox` is smart enough not to install `tox` inside the virtual environments it creates.

Reply:
`tox` shouldn't be in `test/requirements.txt`. The fact that this makes the dev install the requirements into the outer environment unnecessarily is a non-starter for me. Seems like the constraints should be added to the `tox.ini` entries instead?

Reply:
> tox shouldn't be in test/requirements.txt.

I'll remove it.

> Seems like the constraints should be added to the tox.ini entries instead?

I thought I had done that, but apparently I only aded the requirements to `deps` and not the constraints.

Reply:
Oh, I think I see the confusion here. The original docs are suggesting a way to _use_ runner utilizing `mkvirtualenv`. The testing environments are separate, so your change is sort of conflating the two.  A user should be able to _test_ runner without having to activate the environment used to run `ansible-runner` from the command line.

Reply:
Interesting. I missed that.

Reply:
Hmm, it seems like this is in the "Setting up your development environment" section. I don't think it's talking about a virtual environment for using runner. 😕

Reply:
Oh, I see what you mean. the `-e .` is for running from source in addition to being able to run tests. I'm so focused on being able to run tests I forgot about actually running runner.